calico_cni_k8s_test.go

```diff
@@ -30,6 +32,8 @@ import (
 func init() {
 	// Create a random seed
 	rand.Seed(time.Now().UTC().UnixNano())
+	logrus.SetFormatter(&logutils.Formatter{})
```

Comment:
I didn't see context loggers being created in the other test files and since they don't output any log output (at least from the tests I ran), I didn't think it was necessary to set up formatting for them.